compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmPackageGen.java

```diff
@@ -303,13 +307,13 @@ static String getModuleLevelClassName(String orgName, String moduleName, String
 
         if (!moduleName.equals(".")) {
             if (!version.equals("")) {
-                className = cleanupName(version) + "/" + className;
+                className = cleanupName(version) + separator + className;
```

Comment:
If this is always `/` then shall we extract a final field for this, instead of the param.

Reply:
I replaced this to use '.' instead of '/' just for creating the main-class attribute of jar file. Other places use this as '/'.

Reply:
ack.